### PR TITLE
#1146 create rap switcher components

### PR DESCRIPTION
**Pull request by Alex-Work-Account**

### CODE REVIEW

File: packages/ketcher-polymer-editor-react/src/components/rna/Switcher/Switcher.test.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { render, fireEvent, screen } from '@testing-library/react'
+import { Switcher } from './Switcher'
+import { createTheme, ThemeProvider } from '@mui/material/styles'
+import { defaultTheme } from 'styles/theme'
+
+const theme = createTheme(defaultTheme)
+const rnaHandleClick = jest.fn()
+const setup = () =>
+  render(
```
Comment (ViktoriiaFedotkina) [inline packages/ketcher-polymer-editor-react/src/components/rna/Switcher/Switcher.test.tsx:9]: you can use render from 'test-utils', it's already wrapped in themeProvider

File: packages/ketcher-polymer-editor-react/src/components/monomerLibrary/MonomerLibrary.tsx [hunk 2]
```diff
@@ -52,8 +53,8 @@ const MonomerLibrary = () => {
     },
     {
       caption: 'RNA',
-      component: MonomerList,
-      props: { list: [{ name: 'R' }, { name: 'N' }, { name: 'A' }] }
+      component: Switcher
+      // props: { list: [{ name: 'R' }, { name: 'N' }, { name: 'A' }] }
```
Comment (ViktoriiaFedotkina) [inline packages/ketcher-polymer-editor-react/src/components/monomerLibrary/MonomerLibrary.tsx:57]: forgot to remove?

**Review by ViktoriiaFedotkina — APPROVED** [review]

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
tests should be changed

**Review by ViktoriiaFedotkina — APPROVED** [review]

File: packages/ketcher-polymer-editor-react/src/components/rna/Switcher/Switcher.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        lineHeight: '18px',
+        minWidth: '33px',
+        color: isActive ? color.text.light : color.text.dark,
+        ':hover': {
+          backgroundColor: color.button.primary.hover,
+          color: color.text.light
+        }
+      })}
+      onClick={callback}
+    >
```
Comment (karen-sarkisyan) [inline packages/ketcher-polymer-editor-react/src/components/rna/Switcher/Switcher.tsx:49]: It's up to you if you want to change it, but as long as we wanted to minimize inline `css` styles, you could write this component like this:

```tsx
...
import { useTheme, css } from '@emotion/react'
...
const RAPButton = ({ isActive, children, callback }: RAPButtonProps) => {

  const { color } = useTheme()

  const styles = css`
    padding: 3px 12px;
    background-color: ${isActive
      ? color.button.primary.active
      : color.background.canvas};
    border-radius: 8px;
    line-height: 18px;
    min-width: 33px;
    color: ${isActive ? color.text.light : color.text.dark};
    :hover {
      background-color: ${color.button.primary.hover};
      color: ${color.text.light};
    }
  `

  return (
    <Button css={styles} onClick={callback}>
      {children}
    </Button>
  )
}
```

**Review by ElenaOdnoshivkina — APPROVED** [review]

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**